src/main/kotlin/no/nav/dagpenger/inntekt/klassifiserer/KlassifiserInntekt.kt

```diff
@@ -8,8 +9,11 @@ import no.nav.dagpenger.events.inntekt.v1.SpesifisertInntekt
 import java.math.BigDecimal
 import java.time.YearMonth
 
-fun klassifiserOgMapInntekt(spesifisertInntekt: SpesifisertInntekt): Inntekt {
-    val klassifisertePosteringer = klassifiserPosteringer(spesifisertInntekt.posteringer)
+fun klassifiserOgMapInntekt(spesifisertInntekt: SpesifisertInntekt, unleash: Unleash): Inntekt {
+    val klassifisertePosteringer =
+        if (unleash.isEnabled("dp.hyreendring"))
```

Comment:
```suggestion
        if (unleash.isEnabled("dp.hyreendring", false))
```

Legge til en default-verdi for å gjøre det tydeligere hva som skjer om flagget mangler i Unleash?